### PR TITLE
Remove use of deprecated `ALooper_pollAll`

### PR DESCRIPTION
**Pull request by ChrisThrasher**
## Description

An attempt at fixing #3181

### CODE REVIEW

**Comment by eXpl0it3r** [comment]
Would also need to be backported to 2.6.2

File: src/SFML/Window/Android/InputImpl.cpp [hunk 1]
```diff
@@ -166,7 +166,8 @@ void setVirtualKeyboardVisible(bool visible)
 ////////////////////////////////////////////////////////////
 bool isMouseButtonPressed(Mouse::Button button)
 {
-    ALooper_pollAll(0, nullptr, nullptr, nullptr);
+    while (ALooper_pollOnce(0, nullptr, nullptr, nullptr) != ALOOPER_POLL_CALLBACK)
+        ;
```
Comment (eXpl0it3r) [inline src/SFML/Window/Android/InputImpl.cpp:170]: Shouldn't this be the other way around?  
For as long as you get `ALOOPER_POLL_CALLBACK` you should continue processing, every other result should lead to a break.

What's curious is that the official NDK sample, doesn't do it in a loop: https://github.com/android/ndk-samples/pull/1008/files

**Comment by Zombieschannel** [comment]
All errors/warnings are fixed!

**Comment by ChrisThrasher** [comment]
SFMLBuildMaster: Build this please.

**Review by eXpl0it3r — APPROVED** [review]

**Comment by ChrisThrasher** [comment]
SFMLBuildMaster: Build this please.

**Pull request merged**